tasks/assets-size/index.js

```diff
@@ -3,7 +3,7 @@ const assetMap = require('../../dist/mobile-wiki/assets/assetMap').assets;
 
 // set maximum allowed sizes for each asset in kilobytes
 const assetsSizes = {
-  'app.css': 87,
+  'app.css': 90,
```

Comment:
please update your branch
this is now defined in
https://github.com/Wikia/mobile-wiki/blob/dev/config/bundlesize.js

Reply:
@hakubo we'll be merging it to release branch so we'll do it later